Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -8564,10 +8564,8 @@ void HTMLMediaElement::scheduleUpdateMediaState()
         return;
 
     auto logSiteIdentifier = LOGIDENTIFIER;
-    ALWAYS_LOG(logSiteIdentifier, "task scheduled");
```

Comment:
Instead of removing these, can you change them to `INFO_LOG`? This will make them noops unless the log level is increased at runtime.

Reply:
Sure!

Thanks for reviewing!